include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -1156,4 +1157,204 @@ def TT_ExperimentalDescriptorStoreOp : TT_Op<"experimental_descriptor_store", [
     }];
 }
 
+def TT_ExperimentalTensormapAllocOp: TT_Op<
```

Comment:
It seems to me that it not only has the `Alloc` side effect, but also has the `Write` side effect.